tests/integration_test_helper.py

```diff
@@ -96,3 +97,47 @@ def integrate_nodes_helper(cript_api: cript.API, project_node: cript.Project):
     print(my_project_from_api.get_json(sort_keys=False, condense_to_uuid={}, indent=2).json)
     print("==============================================================")
     print("\n\n\n######################################## TEST Passed ########################################\n\n\n")
+
+
+def delete_integration_node_helper(cript_api: cript.API, node_to_delete: UUIDBaseNode) -> None:
+    """
+    1. takes the node/sub-object that needs to be deleted
+    1. checks that it first exists on the API before deleting
+    1. sends an HTTP DELETE to API
+    1. asserts that the API response is 200
+        > This is done under the hood in the `cript.API.delete()`
+        method where at the end it checks that the response is 200, else raises an error
+    1. tries to get the same node via UUID from the API and expects that it should fail
+
+    Notes
+    ------
+    > for future this test should also take the project that the node was in, get the project again,
+    > and compare that the node was successfully deleted/missing from the project
+
+    > within search we can also add assert statements to be sure that the API gave the same node
+    > that we searched for. Not needed at the time of writing this test
+    """
+
+    if not HAS_INTEGRATION_TESTS_ENABLED:
+        pytest.skip("DELETE integration tests with API requires real API and Storage token")
+        return
+
+    # check that the node we want to delete first exists on the API
+    my_node_paginator = cript_api.search(
```

Comment:
I understand the logic.
But would it be better to just send `delete` and handle the error from API if the node doesn't exist?

Reply:
that could streamline the process, but the only thing that we would lose from that would be that we did not prove that the node was there before we went to delete it, so if we start getting bugs we might be asking ourselves, 

`"is it having issues because the API can't delete the node? or is it because the node never existed on the API?"`

I was thinking that writing it this way could be a bit more explicit, but removing the first search would honestly not take away much as far as I can see and would be pretty easy too.

You think remove it?

Reply:
I think, I would insist on getting rid of it in production code, but this is tests only and should be OK.

Reply:
I would honestly be happy getting rid of it too either now or in a future PR. I think both should work and be okay